src/base/datetime/calendar/body/test/language.test.ts

```diff
@@ -0,0 +1,55 @@
+import { expect, fixture } from "@open-wc/testing";
+import { BaseDateTimeCalendarBody } from "../index";
+
+describe("BaseDateTimeCalendarBody", () => {
+  describe("language", () => {
+    it('The first day of the week is "SUN" when not assigning language prop', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      const el = await fixture(container);
+      const item = el.querySelectorAll(
+        ".kuc-base-datetime-calendar-body__header"
+      )[0] as HTMLTableHeaderCellElement;
+      expect(item.innerText).to.equal("SUN");
+    });
+
+    it('The first day of the week is "SUN" when assigning language prop is invalid', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.language = "vn";
+      const el = await fixture(container);
+      const item = el.querySelectorAll(
+        ".kuc-base-datetime-calendar-body__header"
+      )[0] as HTMLTableHeaderCellElement;
+      expect(item.innerText).to.equal("SUN");
+    });
+
+    it('The first day of the week in English is "SUN" when not assigning language prop', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.language = "en";
+      const el = await fixture(container);
+      const item = el.querySelectorAll(
```

Comment:
@nmanhit 
Please using `el.querySelector()` instead of `el.querySelectorAll()` to select first element.
HTMLTableHeaderCellElement has been deprecated, please try other.